### PR TITLE
Enterprise v0.26 release notes + doc changes

### PR DESCRIPTION
**Pull request by jwitz**
Release notes and relevant doc changes for both Enterprise v0.26 and AC 2.2.0. There's a lot of changed files here, but only a few are actually new - the rest are because we cut a new minor docset version. That means you only have to review:

- `/enterprise/v0.26/10_resources/01_release-notes.md`
- `/enterprise/v0.26/04_deploy/02_configure-deployment.md` (For Triggerer information)

There are still some things missing from this PR, mainly:

- Upgrade instructions from v0.25 > v0.26
- LTS support information/ updates

### CODE REVIEW

**Comment by danielhoherd** [comment]
@jwitz to make this PR smaller, you could break out the hyperlink fixes and heading changes into another PR. I think that would make it much easier to review.

**Comment by vishwas-astro** [comment]
This is also a cloud release @jwitz. Do we want to change those release notes as well and not just enterprise release notes?

One big internal thing we are introducing here is the adoption of the OSS based helm chart by the platform. Don't we want to mention anything about that? cc: @aliotta

**Comment by jwitz** [comment]
@vishwas-astro Both this PR and the Cloud release notes PR should have updates based on your latest comments. Could definitely use some additional review on the Verbosity flag release note from @bote795 - will update our CLI reference guide in a separate PR.

File: enterprise/v0.25/06_manage-astronomer/06_upgrade-to-0-26.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            --reset-values \
+            --version $ASTRO_VERSION \
+            --set astronomer.houston.upgradeDeployments.enabled=false \
+            $RELEASE_NAME \
+            astronomer/astronomer
+```
+
+
+
+> **Note:** If you do not specify a patch version above, the script will automatically pull the latest Astronomer Enterprise patch available in the [Astronomer Helm Chart](https://github.com/astronomer/astronomer/releases). If you set `ASTRO_VERSION=0.26` and `--version 0.26`, for example, Astronomer v0.26.4 will be installed if it is the latest v0.25 patch available.
```
Comment (jwitz) [inline enterprise/v0.25/06_manage-astronomer/06_upgrade-to-0-26.md:69]: @aliotta This is the information I added based on our conversation today. Take a look and lmk what you think. Changes related to https://github.com/astronomer/docs/issues/511#event-5576749209 are in a separate PR: https://github.com/astronomer/docs/pull/495

File: enterprise/v0.26/04_deploy/02_configure-deployment.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+To increase the speed at which tasks are scheduled and ensure high-availability, we recommend provisioning 2 or more Airflow Schedulers for production environments. For more information on the Airflow 2.0 Scheduler, refer to Astronomer's ["The Airflow 2.0 Scheduler" blog post](https://www.astronomer.io/blog/airflow-2-scheduler).
+
+### Triggerer
+
+Airflow 2.2 introduces the Triggerer, which is a component for running tasks with [Deferrable Operators](https://airflow.apache.org/docs/apache-airflow/stable/concepts/deferring.html). Like the Scheduler, the Triggerer is highly-available: If a Triggerer shuts down unexpectedly, the tasks it was deferring can be recovered and moved to another Triggerer.
+
+By adjusting the **Triggerer** slider in the Astronomer UI, you can provision up to 2 Triggerers on any Deployment running Airflow 2.2+. To take advantage of the Triggerer's high availability, we recommend provisioning 2 Triggerers for production Deployments.
```
[the remark this answers is not in the document]
Comment (thmzlt) [reply enterprise/v0.26/04_deploy/02_configure-deployment.md:100]: I totally missed this comment (my bad!), but my impression is that the triggerer count has more to do with capacity than availability.

**Comment by vishwas-astro** [comment]
@jwitz this is the first time since the summer of last year we are making a non-LTS minor version of the platform available for enterprise customers.

We may have to modify our upgrades related documentation to reflect the ideas in @astrorfox's version/maintenance policy doc here - https://www.notion.so/astronomerio/Software-Release-and-Maintenance-Policy-d35f8fc358724589bcbc96c50966df2e

0.26.4 is basically a release in the Stable Channel providing access to a new feature - Triggerer. 
0.25.x, 0.23.x and 0.16.x are all in the LTS channel.

We will support upgrades from LTS to LTS versions (eg: 0.16.17 to 0.23.16 OR 0.23.16 to 0.25.11) and these, in all probability, will involve us having custom upgrade scripts, data migration etc. This is usually done with someone from Astronomer being present, so we can debug issues as they arise.

Patch release upgrades within an LTS train (eg: 0.25.4 to 0.25.11) or to the latest within a given stable release (0.26.3 to 0.26.4) is a simple helm upgrade command as documented in our patch upgrade docs.

Upgrade from an LTS to the stable releases between that LTS and the next LTS (eg: upgrade from 0.25.11 to 0.26.4 or 0.25.11 to 0.27.0) should also be a simple helm upgrade command. (there may be a situation where such an upgrade may have associated data migration etc in which case we will have an upgrade script prepared for it)

cc: @danielhoherd @aliotta @andriisoldatenko

File: enterprise/v0.26/10_resources/01_release-notes.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+### Minor Improvements
+
+- You can now create a custom set of cluster-level permissions for the Astronomer Commander service by setting `astronomer.global.clusterRoles: false` in your `config.yaml` file and pushing a new [RoleBinding](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to a [pre-created Kubernetes namespace](/docs/enterprise/v0.26/manage-astronomer/pre-create-namespaces).
+- In the `astronomer.houston.config` section of your `config.yaml` file, you can now configure a list of `allowedSystemLevelDomains []`. If you configure this list, only users with emails from domains specified in the list (for example, `<company>.com`) can be granted System Admin privileges.
+- Greatly improved load times for the **System Admin** page in the UI.
+- In the Astronomer UI, System Admins can now see how many Workspaces each user belongs to from the **Users** tab.
```
Comment (aliotta) [inline enterprise/v0.26/10_resources/01_release-notes.md:46]: This bullet and the following one should not be included. We added the ability for folk to remove these features for performance reasons. Though I'm not sure we want this publicly shared at this time.

File: enterprise/v0.26/10_resources/01_release-notes.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+The flag prints out different levels of logs depending on the value that you pass it. Each possible value (`debug`, `info`, `warn`, `error`, `fatal`, and `panic`) maps to a different Logrus logging level. For more information about these logging levels, read the [Logrus documentation](https://github.com/sirupsen/logrus#level-logging).
+
+### Minor Improvements
+
+- You can now create a custom set of cluster-level permissions for the Astronomer Commander service by setting `astronomer.global.clusterRoles: false` in your `config.yaml` file and pushing a new [RoleBinding](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to a [pre-created Kubernetes namespace](/docs/enterprise/v0.26/manage-astronomer/pre-create-namespaces).
+- In the `astronomer.houston.config` section of your `config.yaml` file, you can now configure a list of `allowedSystemLevelDomains []`. If you configure this list, only users with emails from domains specified in the list (for example, `<company>.com`) can be granted System Admin privileges.
+- Greatly improved load times for the **System Admin** page in the UI.
```
Comment (aliotta) [inline enterprise/v0.26/10_resources/01_release-notes.md:45]: This bullet and the following two should not be included. We added the ability for folk to remove the below two features which leads to the load time improvements. Though I'm not sure we want this publicly shared at this time.

**Comment by jwitz** [comment]
@vishwas-astro This PR should include soft changes related to the new release policy, such as removing mentions of "LTS" from 0.26 upgrade and release note documentation. As for a tangible document, @paolaperaza and I are working on publishing this once 0.26 is out: https://github.com/astronomer/docs/pull/492

This doc should cover the policy changes. Once it's published, we'll strategize over how best to document upgrade workflows but that might not happen until we migrate to docs.astronomer.io

File: enterprise/v0.25/06_manage-astronomer/06_upgrade-to-0-26.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+title: "Upgrade to Astronomer Enterprise v0.26"
+navTitle: "Upgrade to v0.26"
+description: "How to upgrade the Astronomer Enterprise Platform."
+---
+
+## Overview
+
+Astronomer releases are made generally available to Enterprise customers on a quarterly basis as part of a long-term support (LTS) release model.
+
+This guide provides steps for upgrading your Astronomer Enterprise platform from v0.25.x to [v0.26](https://www.astronomer.io/docs/enterprise/v0.26/resources/release-notes), which is the latest available LTS release.
```
Comment (vishwas-astro) [inline enterprise/v0.25/06_manage-astronomer/06_upgrade-to-0-26.md:11]: v0.26.4 is not an LTS release. It is a release in our stable channel, but not an LTS release. To understand difference between and LTS channel and Stable channel, please refer to https://www.notion.so/astronomerio/Software-Release-and-Maintenance-Policy-d35f8fc358724589bcbc96c50966df2e#f70403215b674661ab409ae6415c3eee. 
cc: @astrorfox

**Comment by jwitz** [comment]
@vishwas-astro LTS info should be accurate, plus 0.26 now includes the recently published [release policy doc](https://www.astronomer.io/docs/enterprise/v0.25/resources/release-lifecycle-policy).

File: enterprise/v0.25/06_manage-astronomer/06_upgrade-to-0-26.md (hunk 1 above)
Comment (vishwas-astro) [reply enterprise/v0.25/06_manage-astronomer/06_upgrade-to-0-26.md:67]: @ernest-kr @himabindu07 @danielhoherd - please review the instructions provided here for a 0.25.x version to 0.26.4 upgrade and confirm it looks good.

Comment (vishwas-astro) [reply enterprise/v0.25/06_manage-astronomer/06_upgrade-to-0-26.md:67]: @aliotta do we prefer customers to set astronomer.houston.upgradeDeployments.enabled=true and upgrade their airflow charts to 1.0.7 right away OR do it in a subsequent update of their deployment. Just confirming.

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 6]
```diff
@@ -6,7 +6,7 @@ description: "Astronomer Enterprise release notes."
 
 ## Overview
 
-Astronomer v0.25 is the latest available minor version in our long-term support (LTS) release model for Astronomer Enterprise.
+Astronomer v0.25 is the latest available version in our long-term support (LTS) release model for Astronomer Enterprise. [Astronomer v0.26](/docs/enterprise/v0.26/resources/release-notes). For more information about Enterprise release channels, read [Release and Lifecycle Policies](/docs/enterprise/v0.26/resources/release-lifecycle-policy).
```
Comment (vishwas-astro) [inline enterprise/v0.25/10_resources/01_release-notes.md:9]: I am a little confused here about the reference to Astronomer v0.26

File: enterprise/v0.26/10_resources/01_release-notes.md (hunk 4 above)
Comment (vishwas-astro) [reply enterprise/v0.26/10_resources/01_release-notes.md:45]: @aliotta @jwitz was this addressed?

File: enterprise/v0.26/10_resources/01_release-notes.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+Deferrable operators are a new type of Airflow operator that promises improved performance and lower resource costs. While standard operators and sensors take up a Worker or Scheduler slot even when they are waiting for an external trigger, deferrable operators are designed to suspend themselves and free up that Worker or Scheduler slot while they wait. This is made possible by a new, lightweight Airflow component called the Triggerer.
+
+As part of supporting deferrable operators, you can provision multiple Triggerers on your Astronomer Deployments. By provisioning multiple Triggerers, you can ensure that tasks using Deferrable Operators are run even when one Triggerer goes down. For more information about configuring Triggerers and other resources, see [Configure a Deployment](/docs/enterprise/v0.25/deploy/configure-deployment).
+
+### Minor Improvements
+
+- Greatly improved load times for the **System Admin** page in the UI.
+
+### Bug Fixes
```
[the remark this answers is not in the document]
Comment (vishwas-astro) [reply enterprise/v0.26/10_resources/01_release-notes.md:48]: @jwitz @aliotta was this addressed?

File: enterprise/v0.26/10_resources/05_release-lifecycle-policy.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+It is considered safe to upgrade to minor and patch versions within a major version. Upgrade guidance for major and LTS versions is provided with each release.
+
+## Version Maintenance Policy
+
+The maintenance period for an Astronomer Enterprise version depends on its release channel:
+
+| Release Channel | Frequency of Releases | Maintenance Duration |
+| --------------- | --------------------- | -------------------- |
+| Stable          | Quarterly             | 6 Months             |
```
Comment (vishwas-astro) [inline enterprise/v0.26/10_resources/05_release-lifecycle-policy.md:50]: I thins should be monthly based on the comment above.

File: enterprise/v0.26/10_resources/05_release-lifecycle-policy.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
+---
+title: "Astronomer Release and Lifecycle Policy"
+navTitle: "Release and Lifecycle Policy"
+description: "Astronomer's release and lifecycle policy for Astronomer Enterprise."
+---
+
```
Comment (vishwas-astro) [inline enterprise/v0.26/10_resources/05_release-lifecycle-policy.md:6]: @astrorfox @paolaperaza I would like you two to review this before we publish this and make it visible for our customers.

File: enterprise/v0.26/04_deploy/02_configure-deployment.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (vishwas-astro) [reply enterprise/v0.26/04_deploy/02_configure-deployment.md:100]: I agree with @thmzlt .. @kaxil OR @andrewgodwin do you want to have a final say on this?

File: enterprise/v0.26/10_resources/05_release-lifecycle-policy.md (hunk 9 above)
Comment (jwitz) [reply enterprise/v0.26/10_resources/05_release-lifecycle-policy.md:6]: @paolaperaza Created the initial draft for this document, and we've been working together to get it published here: https://github.com/astronomer/docs/pull/492. I believe that @astrorfox has also been giving feedback on this doc off of GitHub

File: enterprise/v0.23/10_resources/05_release-lifecycle-policy.md [hunk 10]
```diff
@@ -84,4 +84,4 @@ The following table contains the exact lifecycle for each published version of A
 | 0.23               | January 20, 2021 | LTS             | 12 months          | January 2022            |
 | 0.25               | May 11, 2021     | LTS             | 12 months          | May 2022                |
```
Comment (vishwas-astro) [inline enterprise/v0.23/10_resources/05_release-lifecycle-policy.md:85]: In a separate PR, add 0.26.4 as an entry here but in the Stable Release channel with a 6 month maintenance period. Let's go ahead and merge this PR as it has become too big and unwieldy at this time.

File: enterprise/v0.23/10_resources/05_release-lifecycle-policy.md [hunk 11]
```diff
@@ -47,7 +47,7 @@ The maintenance period for an Astronomer Enterprise version depends on its relea
 
 | Release Channel | Frequency of Releases | Maintenance Duration |
 | --------------- | --------------------- | -------------------- |
-| Stable          | Quarterly             | 6 Months             |
+| Stable          | Monthly               | 6 Months             |
 | LTS             | Yearly                | 12 Months            |
```
Comment (vishwas-astro) [inline enterprise/v0.23/10_resources/05_release-lifecycle-policy.md:51]: @astrorfox are we maintaining only 1 LTS revision at any given time then if we are releasing only once a year and then having maintenance period for LTS releases as 12 months?

If this needs changed, let's do it in a separate PR @jwitz and merge this one.

**Review by vishwas-astro — APPROVED** [review]

**Review by paolaperaza — APPROVED** [review]
Alrighty... I think there are 2-3 remaining open questions, but I'm gonna go ahead and merge this since it's such a big PR. Will iterate in future PRs and get clarity from the team on some of those questions.

Thank you @jwitz / @vishwas-astro !
